src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -131,7 +116,7 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
                 })}
```

Comment:
This doesn't seem to be changed, but I get a warning about the lack of keys on the array just above this for keyboard binding?

Reply:
Fixed.